controllers/tables/index.js

```diff
@@ -7,39 +7,44 @@ const _ = require('lodash'); // eslint-disable-line id-length
 const render = require('../../render');
 const Table = require('./model');
 
+const ok = 200;
```

Comment:
For my point of view it is better to make constant varibales as CAPITALIZED. What do you think about it?